### PR TITLE
Change DM grammar source

### PR DESCRIPTION
**Pull request by vvvv-vvvv**
## Description
Changed the DM grammar to a better maintained source that has new keywords

## Checklist:
- [x] **I am changing the source of a syntax highlighting grammar**
  - Old: https://github.com/PJB3005/atomic-dreams
  - New: https://github.com/spacestation13/dm-syntax

### CODE REVIEW

**Review by lildude — CHANGES_REQUESTED** [review]
We can't accept this change. 

Whilst the grammar files are licensed under MIT, the over-all repo is licensed by GPL-3 which means we'll be shipping code that is covered by this license which we can't do.

I'm also not sure GPL3 allows sub-licensing, but that's not relevant as we don't include just the grammar files.

**Comment by vvvv-vvvv** [comment]
Is this fine? The samples for the new extensions are about as complicated as they can get, so they aren't from a real-world project

**Comment by lildude** [comment]
No. We want real world examples as documented. This is used to train the classifier. The `.dme` extension is also not popular enough for addition right now so will only hold back this PR.

**Review by lildude — APPROVED** [review]
LGTM. Thanks.

Note: this PR will not be merged until close to when the next release is made. See [here](https://github.com/github/linguist/blob/master/docs/troubleshooting.md#my-linguist-pr-has-been-merged-but-github-doesnt-reflect-my-changes) for more details.

**Comment by vvvv-vvvv** [comment]
Moved the repo elsewhere

File: vendor/licenses/git_submodule/dm-syntax.dep.yml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ---
-name: atomic-dreams
-version: 14891385ac0110cdcdb22f2dc1a8930cdf501c91
+name: dm-syntax
+version: a63b4388d0df050d40f2c6c6afd85637174abbb5
 type: git_submodule
-homepage: https://github.com/PJB3005/atomic-dreams
+homepage: https://github.com/vvvv-vvvv/dm-syntax.git
```
[the remark this answers is not in the document]
Comment (vvvv-vvvv) [reply vendor/licenses/git_submodule/dm-syntax.dep.yml:5]: I did, and it didn't replace this

**Comment by optimumtact** [comment]
thanks lildude

**Pull request merged**